eZ/Publish/API/Repository/TransactionService.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
+
+use Closure;
+
+interface TransactionService
+{
+    /**
+     * Begin transaction.
+     *
+     * Begins an transaction, make sure you'll call commit or rollback when done,
+     * otherwise work will be lost.
```

Comment:
```suggestion
     * Begins a transaction. Make sure to call commit or rollback when done,
     * otherwise work will be lost.
```

Reply:
Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/164/commits/ffb35ff9c0bd6d94b14a1401cfb71a4e9312cb44